### PR TITLE
[WIP] Mirror Awards For All Pages

### PR DESCRIPTION
**Pull request by davidrapson**
Initial work to mirror Awards For All pages. Re-do of #437 

Tasks to complete:

- [x] Confirm we can reliably determine which content to replace
- [x] Set up A/B test buckets to be used for gradual rollout
- [x] Handle all versions of URLs for awards for all pages
- [ ] Confirm and add real copy for each region (England, Scotland, Wales + Welsh language)
- [ ] Confirm initial percentage to start rollout at

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
         "contrast": "contrastMode",
         "session": "blf-alpha-session",
         "csrf": "_csrf",
-        "abTest": "blf-ab-1-50"
+        "abTestHomepage": "blf-ab-1-50",
+        "abTestAwardsForAll": "blf-ab-afa"
```
Comment (mattandrews) [inline config/default.json]: Worth putting an index on this too to reflect the chances of it being renamed/increased at some point?

File: controllers/toplevel/legacyPages.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+            const additionalText = `
+                <h4>Apply online</h4>
+                <p><br/><a class="roundedButton blueButton" style="display: inline-block;" href="${
+                    applyUrl
+                }">Apply Online</a></p>
+                <h4>Apply by post</h4>
+            `;
+
+            applyTab.innerHTML = additionalText + applyTab.innerHTML;
```
Comment (mattandrews) [inline controllers/toplevel/legacyPages.js]: Is it worth doing a sanity check here to verify we made a change or found the right DOM elements and reporting to Sentry if not? Just to catch the edge case where these pages get modified...

File: config/default.json (hunk 1 above)
Comment (davidrapson) [reply config/default.json]: Not sure I follow?

File: controllers/toplevel/legacyPages.js (hunk 2 above)
Comment (davidrapson) [reply controllers/toplevel/legacyPages.js]: Good shout

**Pull request merged**